activerecord/lib/active_record/migration.rb

```diff
@@ -884,13 +884,14 @@ def method_missing(method, *arguments, &block)
 
     def copy(destination, sources, options = {})
       copied = []
+      schema_migration = options[:schema_migration] || ActiveRecord::SchemaMigration
```

Comment:
I'm still don't get about this `:schema_migration` option.
This option is not used yet in this PR, so any test is not failed even if this option is removed.
Is this option used in GitHub?
If so, should we have a test case for this option?

```diff
diff --git a/activerecord/lib/active_record/migration.rb b/activerecord/lib/active_record/migration.rb
index 9db017cded..a6dd135b5f 100644
--- a/activerecord/lib/active_record/migration.rb
+++ b/activerecord/lib/active_record/migration.rb
@@ -884,7 +884,7 @@ def method_missing(method, *arguments, &block)
 
     def copy(destination, sources, options = {})
       copied = []
-      schema_migration = options[:schema_migration] || ActiveRecord::SchemaMigration
+      schema_migration = ActiveRecord::SchemaMigration
 
       FileUtils.mkdir_p(destination) unless File.exist?(destination)
 
```

Reply:
I will add a test for it.